jobs/resource_job_test.go

```diff
@@ -2466,10 +2478,11 @@ func TestResourceJobUpdate_InstancePoolToNodeType(t *testing.T) {
 					JobID: 789,
 					Settings: &JobSettings{
 						NewCluster: &clusters.Cluster{
-							NodeTypeID:           "node-type-id",
-							DriverNodeTypeID:     "driver-node-type-id",
-							InstancePoolID:       "instance-pool-id-worker",
-							DriverInstancePoolID: "instance-pool-id-driver",
+							AutoterminationMinutes: 60,
+							NodeTypeID:             "node-type-id",
+							DriverNodeTypeID:       "driver-node-type-id",
+							InstancePoolID:         "instance-pool-id-worker",
+							DriverInstancePoolID:   "instance-pool-id-driver",
 						},
```

Comment:
Only change to this file is adding `AutoterminationMinutes` to `60` as that is the default value being pulled in from the registry

Reply:
Is this correct? What does auto-termination even mean for a job cluster? That default seems to make most sense for interactive clusters.

Reply:
Updated, it is no longer there